08-transport.md

```diff
@@ -845,6 +845,25 @@ The responder should produce the given output when fed this input.
     input: 0x00b9e3a702e93e3a9948c2ed6e5fd7590a6e1c3a0344cfc9d5b57357049aa22355361aa02e55a8fc28fef5bd6d71ad0c38228dc68b1c466263b47fdf31e560e139bb
     output: ERROR (ACT3_BAD_TAG)
 
+## Message Encryption Tests
+
+In this test, the initiator sends length 5 messages containing "hello"
+1001 times (we only show 6 example outputs for brevity, and to test
+two key rotations):
+
+	name: transport-message test
+    ck=0x919219dbb2920afa8db80f9a51787a840bcf111ed8d588caf9ab4be716e42b01
+	sk=0x969ab31b4d288cedf6218839b27a3e2140827047f2c0f01bf5c04435d43511a9
+	rk=0xbb9020b8965f4df047e07f955f3c4b88418984aadc5cdb35096b9ea8fa5c3442
+	output 0: 0x5bed0e4d7e2bc28afff2c05dd8fd7a24da81dc17be87e87504e5266a5301529467b98884e0b269
```

Comment:
Actually, my results match up, but only on the latter half of each of the ciphertexts. This means our packet lengths are being encrypted differently.

Can you post your first few serialized packet length's as well (and possibly also nonces)? (in this comment thread)